org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -265,25 +265,24 @@ function resourceSchemaToProviderSchema(resourceId, resourceSchema) {
             }
             break
         default:
-            // Error case
-            console.log("Invalid resourceId: " + resourceId)
```

Comment:
This was the only that was logging so for now made it consistent with the others; The errors today will get picked up and logged at the hubcontroller (higher) layer. Also need to refactor this a bit to allow function here to be able to call into the translator class scoped Logger variable

---
In reply to: [103830390](https://github.com/openT2T/translators/pull/188#discussion_r103830390) [](ancestors = 103830390)